libraries/rainbow/CMakeLists.txt

```
@@ -8,15 +8,7 @@ set(RAINBOW_SOURCES
 src/rainbow.cpp
 )
 
-add_definitions(-DRAINBOW_LIBRARY)
-add_library(PolyMC_rainbow SHARED ${RAINBOW_SOURCES})
-target_include_directories(PolyMC_rainbow PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
+add_library(Launcher_rainbow SHARED ${RAINBOW_SOURCES})
```

```suggestion
add_library(Launcher_rainbow STATIC ${RAINBOW_SOURCES})
```
That should fix the issue